tests/e2e/utils/DriverHelper.ts

```diff
@@ -102,19 +102,42 @@ export class DriverHelper {
                     Logger.error(`DriverHelper.waitVisibility - failed with exception, out of attempts - ${err}`);
                     throw err;
                 }
-                continue;
+
+                if (err instanceof error.TimeoutError) {
+                    if (attempts !== 1) { // waitVisibility was spamming other methods when the number of attempts was 1 - only show message if attempts > 1
```

Comment:
DriverHelper.waitVisibility was spamming other methods with messages when the timeout = DEFAULT_POLLING
only display trace message when attempts > 1